### PR TITLE
resolve documentation build issues

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -40,6 +40,7 @@ opencv-python
 Pillow>=8.3.2
 matplotlib>=3.4,<3.5.3
 scipy
+nibabel
```
Comment (adrianboguszewski) [inline requirements.txt:43]: This is required only for 110. I'd prefer to install it inside 110 instead of adding a new requirement for all.

**Review by adrianboguszewski — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (eaidova) [reply requirements.txt]: moved

File: notebooks/110-ct-segmentation-quantize/pytorch-monai-training.ipynb [hunk 2]
```diff
@@ -35,6 +35,15 @@
...
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install nibabel"
+   ]
```
Comment (adrianboguszewski) [inline notebooks/110-ct-segmentation-quantize/pytorch-monai-training.ipynb:45]: Should it be added to that notebook, not data-preparation-ct-scan.ipynb? Nibabel is imported there

File: .github/workflows/convert_notebooks.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
     - 'main'
+    - 'ea/docs_fix' # TODO remove after test
```
Comment (adrianboguszewski) [inline .github/workflows/convert_notebooks.yml:9]: Should it be removed now or after the merge?

Comment (eaidova) [reply .github/workflows/convert_notebooks.yml:9]: I remove it before merge, I just want to make sure that conversion build passed, when it succeed, I remove

File: notebooks/110-ct-segmentation-quantize/pytorch-monai-training.ipynb (hunk 2 above)
Comment (eaidova) [reply notebooks/110-ct-segmentation-quantize/pytorch-monai-training.ipynb:45]: we do not use data-preparation notebook for docs buiilding. I see only that this notebook filed in builds. It failed during some internal imports from monai

Comment (adrianboguszewski) [reply notebooks/110-ct-segmentation-quantize/pytorch-monai-training.ipynb:45]: I see

**Pull request merged**